### ISSUE TITLE
[SF][A11Y][MultiComboBox]Cannot see all options of multicombox by pressing Enter 

### ISSUE THREAD

**Issue by yuyuCream** [SAP/ui5-webcomponents#5393]
### **Bug Description**
Cannot see all options of multicombox by pressing Enter.

### **Expected Behavior**
Press Enter key and user can see all options of multi combobox.

### **Steps to Reproduce**
1. Go to https://sapui5.hana.ondemand.com/#/entity/sap.m.MultiComboBox/sample/sap.m.sample.MultiComboBox
2. Press Tab to focus the multicombobox
3. Press Enter, and nothing happen

### **Context**
- UI5 Web Components version: 1.4.0
- OS/Platform: Mac
- Browser:Chrome
- Affected component: MultiComboBox

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Business impact: {...}

### **Additional context**
I am from SAP Successfactors. And issues are for b2211 released project.

**Comment by tsanislavgatev** [comment]
Hello @yuyuCream, 
The provided URL is to openui5 demokit.
This is the MultiComboBox in ui5-webcomponents: https://sap.github.io/ui5-webcomponents/playground/components/MultiComboBox/

Also, by specification the open all options functionality is triggered by either Alt + ArrowDown or F4 button.

The Enter button is to confirm selection or input.

Best Regards,
Tsanislav
